Scripts/MicrosoftTeamsAsk/MicrosoftTeamsAsk.yml

```diff
@@ -17,40 +17,46 @@ args:
   secret: false
 - default: false
   defaultValue: 'yes'
-  description: First option for the team member reply.
+  description: First reply option.
   isArray: false
   name: option1
   required: false
   secret: false
 - default: false
   defaultValue: 'no'
-  description: Second option for the team member reply.
+  description: Second reply option.
   isArray: false
   name: option2
   required: false
   secret: false
 - default: false
-  description: A comma delimited list of additional options (in case more than 2 options
-    are needed)
+  description: A CSV list of additional options (in case more than 2 options
+    are required).
   isArray: false
   name: additional_options
   required: false
   secret: false
 - default: true
-  description: Team member to ask
+  description: Team member to which to send the question.
   isArray: false
   name: team_member
-  required: true
+  required: false
   secret: false
 - default: false
-  description: Playbook Task ID close with the reply. If none provide, no playbook
+  description: Task ID of the playbook task to close with the reply. If not provided, no playbook
     task will be closed.
   isArray: false
   name: task_id
   required: false
   secret: false
-comment: Ask a user a question on Microsoft Teams and expect a response. The response
-  can also close a task (might be conditional) in a playbook.
+- default: false
+  description: Channel to which to send the question.
+  isArray: false
+  name: channel
+  required: false
+  secret: false
+comment: Send a user a question with predefined response options on Microsoft Teams. The response
```

Comment:
user or channel
also, be consistent: "team member" or "user"

Reply:
@yaakovi Absolutely right. Let's go with "team member"

Send a team member or channel a question with predefined....